app/assets/javascripts/live_search.js

```diff
@@ -45,73 +56,94 @@
     this.focusErrorMessagesOnLoad(this.$form)
 
     // prevent page refresh on search submit button click
-    // instead trigger the AJAX results fetch
-    this.$searchSubmitButton.on('click',
-      function (e) {
+    // instead trigger the results fetch
+    if (this.$searchSubmitButton) {
+      this.$searchSubmitButton.addEventListener('click', function (e) {
         e.preventDefault()
         this.formChange(e)
```

Comment:
Looking at the `formChange` function, I don't think it actually does anything with the event variable `e` passed into it. Could it be removed?

https://github.com/alphagov/finder-frontend/blob/c5940fdeacadb5f21b843c41904020b04e8dbfb8/app/assets/javascripts/live_search.js#L168-L177

Reply:
Good point, seems not, have removed.